vnext/Scripts/rnw-dependencies.ps1

```diff
@@ -515,63 +507,20 @@ $requirements = @(
         Name = ".NET SDK (LTS, = $dotnetver)";
         Tags = @('appDev');
         Valid = { CheckDotNetCore; };
-        Install = { ChocoInstall dotnet-$dotnetver-sdk };
-        InstallsWithChoco = $true;
+        Install = { WinGetInstall Microsoft.DotNet.SDK.$wingetDotNetVer };
         HasVerboseOutput = $true;
     }
 );
 
-$ChocoCmd = ""; # This should get set when EnsureChocoForInstall is called
-
-function EnsureChocoForInstall {
```

Comment:
Do we have a solid min version of Windows 10 that includes winget by default? We currently specify 10.0.16299.0 as the min version to dev RNW.

Reply:
👍 I'll look at adding some verification that winget is available and provide feedback if its not.

Reply:
Ensure that feedback includes enough instructions for the user to both get winget installed and run it at least once to get past the license agreement